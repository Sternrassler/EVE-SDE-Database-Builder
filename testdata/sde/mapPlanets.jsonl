{"planetID":40000001,"solarSystemID":30000001,"planetName":"Tanoo I","typeID":2016,"x":1.5e11,"y":0.0,"z":0.0,"celestialIndex":1,"radius":6000000.0}
{"planetID":40000002,"solarSystemID":30000142,"planetName":"Jita IV","typeID":2016,"x":2.0e11,"y":1.0e10,"z":-5.0e10,"celestialIndex":4,"radius":7000000.0}
{"planetID":40000003,"solarSystemID":30000003,"planetName":"H-PA29 III","typeID":2063,"x":-3.0e11,"y":2.0e11,"z":1.0e11,"celestialIndex":3,"radius":5500000.0}
