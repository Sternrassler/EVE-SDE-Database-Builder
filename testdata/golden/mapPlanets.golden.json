[
  {
    "planetID": 40000001,
    "planetName": "Tanoo I",
    "solarSystemID": 30000001,
    "typeID": 2016,
    "x": 150000000000,
    "y": 0,
    "z": 0
  },
  {
    "planetID": 40000002,
    "planetName": "Jita IV",
    "solarSystemID": 30000142,
    "typeID": 2016,
    "x": 200000000000,
    "y": 10000000000,
    "z": -50000000000
  },
  {
    "planetID": 40000003,
    "planetName": "H-PA29 III",
    "solarSystemID": 30000003,
    "typeID": 2063,
    "x": -300000000000,
    "y": 200000000000,
    "z": 100000000000
  }
]